pootle/apps/import_export/management/commands/export.py

```diff
@@ -40,33 +55,42 @@ def _create_zip(self, stores, prefix):
         self.stdout.write("Created %s\n" % (f.name))
 
     def handle_all(self, **options):
-        project_query = Project.objects.all()
-
-        if self.projects:
-            project_query = project_query.filter(code__in=self.projects)
-
         if options['pootle_path'] is not None:
+            if options['export_tmx']:
+                raise CommandError("--path option can't be used with --tmx option")
```

Comment:
I think this can be achieved with https://docs.python.org/3/library/argparse.html#mutual-exclusion